src/requirements.yml

```diff
@@ -3,6 +3,10 @@
   name: amazon_efs_utils
 - src: https://github.com/cisagov/ansible-role-amazon-ssm-agent
   name: amazon_ssm_agent
+- src: https://github.com/cisagov/ansible-role-automated-security-updates
+  name: automated_security_updates
+- src: https://github.com/cisagov/ansible-role-backports
+  name: backports
```

Comment:
Looks like this got re-added after being removed in https://github.com/cisagov/docker-packer/commit/6dbd0c4181c74ebaa46d3682622351d439c74921.

```suggestion
```